jenkins/gradle/gradle-check.jenkinsfile

```diff
@@ -150,28 +152,38 @@ pipeline {
                         def invokedBy
                         def pullRequest
                         def pullRequestTitle
+                        def gitReference
+                        def pullRequestOwner
                         switch (true) {
                             case env.BUILD_CAUSE.contains('Started by user'):
                                 invokedBy = 'User'
-                                pullRequest = "${GIT_REFERENCE}"
-                                pullRequestTitle = "Null"
+                                pullRequest = "null"
+                                pullRequestTitle = "null"
+                                gitReference = "${GIT_REFERENCE}"
+                                pullRequestOwner = "null"
                                 break
                             case env.BUILD_CAUSE.contains('Started by timer'):
                                 invokedBy = 'Timer'
-                                pullRequest = "${GIT_REFERENCE}"
-                                pullRequestTitle = "Null"
+                                pullRequest = "null"
+                                pullRequestTitle = "null"
+                                gitReference = "${GIT_REFERENCE}"
+                                pullRequestOwner = "null"
                                 break
-                            case "${pr_number}" == "Null":
+                            case "${post_merge_action}" == "true":
```

Comment:
Same as above.
With this change https://github.com/opensearch-project/OpenSearch/pull/13786/files the `pr_number` is not null anymore. A new field `post_merge_action` with true/false is added to identify the `push` event.